### DISCUSSION TITLE
Hubitat timer app question

### DISCUSSION THREAD

**Discussion by gitstevenr** [community/community#86504] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am using a timer app in my Hubitat system that was created by github user botched1.
App name is Switch Timer with reset.
The timer app has 2 preferences, 'Delay before On/Off' and 'Reset'.
I'm not sure what reset does.  When I change the setting it doesn't appear to do anything.

![image](https://github.com/community/community/assets/74972584/b834f637-a62d-4155-b4f6-102b43d0146b)

**Comment by AidenJamesYt** [comment]
For this I would ask the person that made it


here is their profile: https://github.com/botched1


hope it helps :)

**Reply by gitstevenr** [reply]
Thanks.  
I have his/her profile. But I don't see how to send a message directly to them.
I searched how to send a direct message and it appears that function was removed at some point.

**Reply by AidenJamesYt** [reply]
honestly, you most likely wont be able to get in touch with them as they dont have any public profile information. if you can find their site (if they have one), then you may be able to contact them through there

**Reply by gitstevenr** [reply]
Thanks.

**Reply by AidenJamesYt** [reply]
np
